app/views/annotations/_annotations_overview_accordion.html.erb

```diff
@@ -0,0 +1,72 @@
+<% accordionLabel = "annotations-overview-accordion" %>
+<% accordionLabel = "student-#{accordionLabel}" if is_students_annotations %>
+<div class="accordion" id="<%= accordionLabel %>">
+
+  <% annotations_by_lecture.each_with_index do |(lecture, annotations), index| %>
+    <div class="accordion-item">
+      <% collapseLabel="annotation-overview-collapse-#{index}-#{is_students_annotations}" %>
+
+        <!-- Accordion header -->
+        <h2 class="accordion-header">
+          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+              data-bs-target="#<%= collapseLabel %>"
+              aria-expanded="true" aria-controls="#<%= collapseLabel %>">
+            <%= lecture %>
```

Comment:
I think it would be nice to display how many annotations there are for this lecture (this might prevent one from needlessly opening the accordion).